utils/notifications.js

```diff
@@ -6,8 +6,11 @@ const {getResponseJSON, setHeadersDomainRestricted, setHeaders, logIPAdddress, r
 const {getScheduledNotifications, saveNotificationBatch} = require("./firestore");
 const {getParticipantsForNotificationsBQ} = require("./bigquery");
 const conceptIds = require("./fieldToConceptIdMapping");
+let twilioClient, messagingServiceSid;
 
-const [twilioClient, messagingServiceSid] = setupTwilioSms();
+(async () => {
```

Comment:
This can still lead to a race condition because an async function is still wrapped in a promise and the await inside doesn't block processing.  The safe way to handle this would be to run the setupTwilioSms function with an await in front of it inside the functions that are going to use the twilio client.

Reply:
Good suggestion, and notes taken. I'll make such change next time when updating this repo.

Reply:
Regarding possible race condition, based on long procedures involved in notification sending,  I believe `setupTwilioSms` call will be finished before `twilioClient` is used.

Reply:
I thought about this issue again. In Node microtask queue, `setupTwilioSms` call is way ahead of `sendSmsBatch` call, thus I believe race condition won't occur.